src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1140,10 +1132,29 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 			},
 
 			onDrop: e => {
-				tab.classList.remove('dragged-over');
-				this.updateDropFeedback(tab, false, tabIndex);
+				this.updateDropFeedback(tab, false, e, tabIndex);
+
+				// compute the target index
+				let targetIndex = tabIndex;
```

Comment:
A comment would help here what the rules are for finding the index, including the lengthy computation for when there are `editorIdentifiers`